### PR TITLE
add permissions for clinvar ingest deployment

### PR DESCRIPTION
**Pull request by bpblanken**

### CODE REVIEW

File: terraform/shared/iam/main.tf [hunk 1]
```diff
@@ -194,3 +194,49 @@ resource "google_service_account_iam_member" "prod_cloudbuild_cloudfunc_binding"
   role               = "roles/iam.serviceAccountUser"
   member             = "serviceAccount:974091131481@cloudbuild.gserviceaccount.com"
 }
+
+# The ServiceAccount that the external-secrets controller will use to identify itself to the GCP API.
```
[the remark this answers is not in the document]
Comment (bpblanken) [reply terraform/shared/iam/main.tf:198]: bad copy paste

**Review by theferrit32 — APPROVED** [review]

**Comment by larrybabb** [comment]
@bpblanken Looks like the ball is in your court to do the merge unless there is a good reason we are holding off. if so, please add a comment so we can keep things fresh. thanks.

**Comment by bpblanken** [comment]
@theferrit32 do you want to re-look at this?  I added a bunch of stuff yesterday to get it working... but don't think there's anything particularly noteworthy!

**Comment by sjahl** [comment]
@bpblanken I'll take a look -- some applied changes from this PR are also preventing me from applying a different PR

**Review by sjahl — APPROVED** [review]
this looks good to me -- i'll resolve the merge conflict

**Pull request merged**